model/lifecycle_test.go

```diff
@@ -732,15 +732,15 @@ func TestCreateBuildFromVersion(t *testing.T) {
 			},
 		}
 
-		smallContainerSize := ContainerResources{
-			MemoryMB: 128,
-			CPU:      256,
-		}
 		pref := &ProjectRef{
 			Id:         "projectId",
 			Identifier: "projectName",
-			ContainerSizes: map[string]ContainerResources{
-				"small": smallContainerSize,
+			ContainerSizeDefinitions: []ContainerResources{
```

Comment:
If there may be multiple, can we please add more to the array?

Reply:
Sure